### PR TITLE
Volunteers: Add index, edit, show views

### PR DESCRIPTION
**Pull request by tpham27**

### CODE REVIEW

File: app/controllers/volunteers_controller.rb [hunk 1]
```diff
@@ -2,22 +2,59 @@ class VolunteersController < ApplicationController
...
+    redirect_to @volunteer
   end
 
+
   def edit
+  	@volunteer = Volunteer.find(params[:id])
+  end
 
+  def patch
+  	binding.pry
```
Comment (BintLopez) [inline app/controllers/volunteers_controller.rb]: @tpham27 -- can you please remove this binding?

Comment (tpham27) [inline app/controllers/volunteers_controller.rb]: yes will do!

File: app/views/volunteers/index.html.erb [hunk 2]
```diff
@@ -1,3 +1,47 @@
...
+        <% end %>
+        <%= link_to(volunteer, class: "btn btn-small btn-danger", method: :delete)  do %>
+          Destroy
+        <% end %>
+      </td>
+      </tr>
+  <% end %>
+<tr>
+  <% else %>
+    <td>Move along nothing to see here<td>
```
Comment (BintLopez) [inline app/views/volunteers/index.html.erb]: Lol I <3 it.

Comment (tpham27) [inline app/views/volunteers/index.html.erb]: :smile:

**Pull request merged**